synthesizer/program/src/mapping/key/mod.rs

```diff
@@ -15,27 +15,16 @@
 mod bytes;
 mod parse;
 
-use console::{
-    network::prelude::*,
-    program::{Identifier, PlaintextType},
-};
+use console::{network::prelude::*, program::PlaintextType};
 
-/// A key statement is of the form `key {name} as {plaintext_type}.public`.
+/// A key statement is of the form `key as {plaintext_type}.public`.
 #[derive(Clone, PartialEq, Eq, Hash)]
```

Comment:
does `Hash` still hold without the `Identifier`? I'm not too familiar with the related objects, but the name `PlaintextType` suggests to me that they might be some overlap between them?

Reply:
It turns out we actually don't rely on it at all.